Source/WebKit/UIProcess/API/ios/WKWebViewIOS.h

```diff
@@ -68,7 +68,7 @@ enum class TapHandlingResult : uint8_t;
 - (UIView *)_currentContentView;
 
 - (void)_didCommitLoadForMainFrame;
-- (void)_didCommitLayerTree:(const WebKit::RemoteLayerTreeTransaction&)layerTreeTransaction mainFrameData:(const std::optional<WebKit::MainFrameData>&)mainFrameData;
+- (void)_didCommitLayerTree:(const WebKit::RemoteLayerTreeTransaction&)layerTreeTransaction mainFrameData:(const std::optional<WebKit::MainFrameData>&)mainFrameData pageData:(const WebKit::PageData&)pageData transactionID:(const WebKit::TransactionID&)transactionID;
```

Comment:
Could just pass the whole bundle as a param here, since that's pretty much all of the components now.